Google.Api.Gax.Tests/Json/JsonTokenizerTest.cs

```diff
@@ -120,8 +120,10 @@ public void NumberValue(string json, double expectedValue)
         [InlineData("1e-")]
         [InlineData("--")]
         [InlineData("--1")]
-        [InlineData("-1.7977e308")]
-        [InlineData("1.7977e308")]
+        // TODO: Investigate these when we actually start using the JSON code.
```

Comment:
This is and AssertThrows, so the test failing means that this number format is now valid, which is not a bad thing.

I think it's OK to remove these two cases.

Reply:
I think it's a bad thing if the numbers can't actually be represented due to overflow (which I believe was the issue). I'd like to look more closely at this later, so I'll leave them in if that's okay.